spec/resources/numismatics/coins/numismatics/coins_controller_spec.rb

```diff
@@ -75,7 +75,7 @@
     end
     let(:resource) { FactoryBot.create_for_repository(:coin) }
     it "retrieves an array of facet values to for use in populating select boxes" do
-      metadata_adapter = Valkyrie::MetadataAdapter.find(:index_solr)
+      metadata_adapter = Valkyrie::MetadataAdapter.find(:indexing_persister)
```

Comment:
does this mean we haven't been persisting changes to coins?

Reply:
The actual controller persists to both places, this test just tried to be more efficient by only persisting to Solr (which we can't query now.)